recipe/magento2.php

```diff
@@ -4,6 +4,8 @@
 require_once __DIR__ . '/common.php';
 
 // Configuration
+set('static-content-locales', 'en_US');
```

Comment:
Please, use _ underscore in config name.

Reply:
done.